helm/defectdojo/values.yaml

```diff
@@ -124,6 +124,7 @@ celery:
       limits:
         cpu: 2000m
         memory: 256Mi
+    revisionHistoryLimit: 10
```

Comment:
Can we simplify helm to use same settings for the all deployments, like in root level but applicable for all deployments.

Reply:
Sure, why not?

Reply:
I updated the chart to support both.